docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -64,7 +64,7 @@ You can upload a custom checkout to your store's server using WebDAV. The instru
 4. Navigate into the `/dist` folder in your checkout project.
 5. Copy the contents of the `/dist` folder and paste it into the checkout folder.
 
-## Installing a custom checkout  
+## Creating a custom checkout  
```

Comment:
Does this section provide steps on creating a custom checkout?

Reply:
Since this tutorial doesn't provide instructions on how to develop a custom checkout, I think "installing" is more accurate here.

Reply:
OK, do you have an another idea for how to fix the anchor?